apps/src/p5lab/P5Lab.js

```diff
@@ -740,6 +740,11 @@ P5Lab.prototype.afterInject_ = function(config) {
   );
 
   if (this.isSpritelab) {
+    if (this.level.blocklyVariables) {
```

Comment:
could this be decoupled from the `this.isSpritelab` conditional on line 742? right now, `blocklyVariables` will only exist on spritelab levels, but we would need to update this code if we were to use this field on other level types, and the concepts are unrelated enough that i think they should be treated separately